packages/headless/src/features/facets/range-facets/date-facet-set/date-facet-set-slice.test.ts

```diff
@@ -141,6 +143,16 @@ describe('date-facet-set slice', () => {
     );
   });
 
+  it('#updateDateFacetRangeAlgorithm updates #DateFacetRequest.rangeAlgorithm to the correct value', () => {
+    state['1'] = buildMockDateFacetRequest();
+    const action = updateDateFacetRangeAlgorithm({
+      facetId: '1',
+      rangeAlgorithm: 'even',
+    });
+    const finalState = dateFacetSetReducer(state, action);
+    expect(finalState['1'].rangeAlgorithm).toBe('even');
```

Comment:
Related to the comment below on extracting the logic into a handler, in the date and numeric slice test files, we would just need to check that the correct handler is called.